src/app/projects/components/project-metadata-form/project-metadata-form.component.ts

```diff
@@ -124,7 +124,13 @@ export class ProjectMetadataFormComponent implements OnInit, OnDestroy {
 
   onSave({ valid, validationSkipped, value }) {
     if (!this.incrementProjectTab()) {
-      if (valid || validationSkipped) {
+      if (valid) {
```

Comment:
In lines 136-140, it looks like there's a redundant open and closing bracket for else, not part of the change but if you could fix it, it would be nice.

Reply:
There's no existing unit test for onSave but it would be nice to also add a unit test for `onSave` function , if you need help on it, let me know!

Reply:
I removed the redundant brackets.
Can you commit the unit test you are suggesting? since I have no experience with node / typescript unit tests.